pkg/apis/integreatly/v1alpha1/grafanadatasource_types.go

```diff
@@ -138,6 +143,9 @@ type GrafanaDataSourceSecureJsonData struct {
 	HTTPHeaderValue7 string `json:"httpHeaderValue7,omitempty"`
 	HTTPHeaderValue8 string `json:"httpHeaderValue8,omitempty"`
 	HTTPHeaderValue9 string `json:"httpHeaderValue9,omitempty"`
+	// Fields for Stackdriver data sources
+	PrivateKey string `json:"privateKey,omitempty"`
+	Version    string `json:"version,omitempty"`
```

Comment:
I think the 'version' field is outside of the 'secureJsonData' object, so I don't think it needs to be included here.

Reply:
Oh true, thanks.